ndbench-web/src/main/java/com/netflix/ndbench/defaultimpl/InjectedWebListener.java

```diff
@@ -48,7 +49,7 @@ protected Injector getInjector()
         moduleList.add(new NdBenchGuiceModule());
         moduleList.add(new NdBenchClientModule());
         moduleList.add(new ArchaiusModule()); //Archaius-2
-
+        moduleList.add(new NfndbenchEsModule());
```

Comment:
Let us assume that somebody wants to use the Cassandra plugin. Why would he want to add the ES module?